utils/find_actors.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+import ast
+import os
+
+
+def is_direct_actor_def(ast_node):
+    if not isinstance(ast_node, ast.ClassDef):
+        return False
+
+    has_actor_base = False
+    for class_base in ast_node.bases:
+        # We are looking for direct name 'Actor'
+        if not isinstance(class_base, ast.Name):
+            continue
+
+        if class_base.id == 'Actor':
+            has_actor_base = True
+            break
+
+    return has_actor_base
+
+
+def extract_actor_name_from_def(actor_class_def):
+    actor_name = None
+    for child in actor_class_def.body:
+        if not isinstance(child, ast.Assign):  # We are looking for `name = 'name'`
+            continue
+        assignment = child
+        for target in assignment.targets:
+            if isinstance(target, ast.Name) and target.id == 'name' and isinstance(assignment.value, ast.Constant):
+                lhs = assignment.value
+                actor_name = lhs.value
+                break
+        if actor_name is not None:
+            break
+    return actor_name
+
+
+def get_actor_names(actor_path):
+    with open(actor_path) as actor_file:
+        actor_def = ast.parse(actor_file.read())
+        actor_defs = [ast_node for ast_node in actor_def.body if is_direct_actor_def(ast_node)]
+        actors = [extract_actor_name_from_def(actor_def) for actor_def in actor_defs]
+    return actors
+
+
+def make_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('actor_names', nargs='+', help='Actor names to look for')
```

Comment:
Could you please add a reference, that actor name means the `name` attribute of an actor object?

Reply:
Okayo, done.